common/src/main/java/whocraft/tardis_refined/common/items/ScrewdriverMode.java

```diff
@@ -0,0 +1,9 @@
+package whocraft.tardis_refined.common.items;
+
+public enum ScrewdriverMode {
+
+    GENERIC,
```

Comment:
What is the difference between them all? Javadoc the enum

Reply:
Added documentation